misk/web/@misk/common/src/index.ts

```diff
@@ -1,2 +1,32 @@
 export * from "./externals"
-export * from "./interfaces"
\ No newline at end of file
+
+/**
+ * Common Interfaces
+ */
+import { IconName } from "@blueprintjs/icons"
+interface IMiskAdminTab {
+  icon: IconName
+  name: string
+  slug: string
+  url_path_prefix: string
+}
+
+interface IMiskAdminTabs {
+  [key:string]: IMiskAdminTab
+}
+
+interface IMiskWindow extends Window {
+  __REDUX_DEVTOOLS_EXTENSION_COMPOSE__: any
+  Misk: {
+    Binder: any
+    Common: any
+    Components: any
+    History: any
+  }
+  MiskTabs: {
```

Comment:
Good catch! The `Misk` is still there but there is a separate `MiskTabs` that exists on Window but wasn't part of the previous code since it wasn't needed in that file. Since this interface will be common to all of them, I added all of the objects that Misk related libraries could bind to the Window so the interface can be used everywhere.